snippets/quarkus.json

```diff
@@ -0,0 +1,78 @@
+{
+	"Quarkus - new resource class": {
+		"prefix": "qrc",
+		"body": [
+			"package ${1:packagename};",
+			"",
+			"import javax.ws.rs.GET;",
+			"import javax.ws.rs.Path;",
+			"import javax.ws.rs.Produces;",
+			"import javax.ws.rs.core.MediaType;",
+			"",
+			"@Path(\"${2:path}\")",
+			"public class ${TM_FILENAME_BASE} {",
+			"",
+			"\t@GET",
+			"\t@Produces(MediaType.TEXT_PLAIN)",
+			"\tpublic String ${3:methodname}() {",
+			"\t\treturn \"hello\";",
+			"\t}",
+			"}"
+		],
+		"description": "Create a new Quarkus resource class"
+	},
+	"Quarkus - new resource method": {
+		"prefix": "qrm",
+		"body": [
+			"\t@GET",
+			"\t@Produces(MediaType.TEXT_PLAIN)",
+			"\tpublic String ${1:methodname}() {",
+			"\t\treturn \"hello\";",
+			"\t}",
+			"}"
```

Comment:
Why is there an extra "}" here?

Reply:
nice catch. Fixed (along with the unnecessary indentation)